app-text/libgepub/libgepub-0.7.0.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org meson xdg
+
+DESCRIPTION="GObject based library for handling and rendering epub documents"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/libgepub"
+
+LICENSE="LGPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE="+introspection"
+
+RDEPEND="
+	app-arch/libarchive
+	dev-libs/glib:2
+	dev-libs/libxml2
+	net-libs/libsoup:3.0
```

Comment:
You removed the webkit-gtk dependency. Did you check the `pkgdiff -b` output?

````diff
@@ -31,8 +31,7 @@ libversion = '@0@.@1@.@2@'.format(sovers
 cc = meson.get_compiler('c')
 
 gepub_deps = [
-  dependency('webkit2gtk-4.0'),
-  dependency('libsoup-2.4'),
+  dependency('libsoup-3.0'),
   dependency('glib-2.0'),
   dependency('gobject-2.0'),
   dependency('gio-2.0'),
@@ -40,15 +39,29 @@ gepub_deps = [
   dependency('libarchive')
 ]
 
+if get_option('widget')
+  webkit2gtk = dependency('webkit2gtk-4.1')
+  gepub_deps += [webkit2gtk]
+endif
+
````

There's a new `widget` option.

Reply:
Added the widget option as new USE flag in next commit